VMEncryption/main/ExtensionParameter.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, hutil, logger, distro_patcher, encryption_environment, protec
         self.KeyVaultURL = public_settings.get(CommonVariables.KeyVaultURLKey)
         self.AADClientID = public_settings.get(CommonVariables.AADClientIDKey)
         self.AADClientCertThumbprint = public_settings.get(CommonVariables.AADClientCertThumbprintKey)
+        self.KeyVaultResourceId = public_settings.get(CommonVariables.KeyVaultResourceIdKey)
+        self.KekVaultResourceId = public_settings.get(CommonVariables.KekVaultResourceIdKey)
```

Comment:
can we add the migrate param here? it can then be logged/identified easily.

Reply:
yes -- having the migrate param in the logs would be useful.

Reply:
Added.